clients/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metaplex-foundation/mpl-toolbox",
-  "version": "0.9.2",
+  "version": "0.9.3",
```

Comment:
I think we want this to be 0.10.0 so downstream packages don't pull these changes.

Reply:
https://github.com/metaplex-foundation/mpl-toolbox/commit/e8306418ea63a8dd4003419e8691a83719f95eb9